components/signup/signup_email/signup_email.test.tsx

```diff
@@ -4,24 +4,24 @@
 import React from 'react';
 import {shallow} from 'enzyme';
 
-import SignupEmail from 'components/signup/signup_email/signup_email.jsx';
+import {UserProfile} from 'mattermost-redux/types/users';
+
+import SignupEmail from 'components/signup/signup_email/signup_email';
+import {PasswordConfig} from 'components/signup/signup_email/index';
 
 import {browserHistory} from 'utils/browser_history';
 
 describe('components/SignupEmail', () => {
     const requiredProps = {
         location: {
-            query: {
-                token: '9f392f193973g11ggh398h39hg0ghH',
-                email: 'test@example.com',
-            },
+            search: '?token=9f392f193973g11ggh398h39hg0ghH&email=test@example.com',
```

Comment:
Good catch on this. I guess this test didn't work before since it was an invalid object.